src/csv_decode.rs

```diff
@@ -15,9 +15,9 @@
 // along with this program. If not, see
 // <https://www.gnu.org/licenses/>.
 
-//! CSV decoder to read BinaryEye app's "CSV with semicolon delimiter"
+//! CSV decoder to read `BinaryEye` app's "CSV with semicolon delimiter"
 //! export format. Inspired by
-//! https://docs.rs/csv/1.1.6/csv/#example-with-serde
+//! <https://docs.rs/csv/1.1.6/csv/#example-with-serde>
```

Comment:
Am assuming those were manual edits. Were you doing this checking `cargo doc` every time, or was there a trick tool I don't know about?

Reply:
`cargo clippy` showed this as a warning